.pre-commit-config.yaml

```diff
@@ -32,7 +32,7 @@ repos:
     hooks:
       - id: ESLint
         name: ESLint
-        entry: npx eslint -c .eslintrc.yaml
+        entry: npx eslint -c .eslintrc.yaml --fix
```

Comment:
what does this do?

Reply:
[https://eslint.org/docs/latest/user-guide/command-line-interface#--fix](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix)
Automatically fixes linting problems. I found it helpful when there's like 80+ linting errors that would be a pain to fix manually. I guess one concern would be if the automatic linter breaks the code in some way, but maybe we could try adding the flag for a little bit and see if there are any major issues?

Reply:
Yeah that sounds fine